stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/HomeScreen.kt

```diff
@@ -140,29 +139,26 @@ fun HomeScreen(
     }
 }
 @Composable
-fun checkVisibility(viewModel: HomeViewModel, manageStockViewModel: ManageStockViewModel): Boolean {
+fun checkVisibility(viewModel: HomeViewModel): Boolean {
```

Comment:
you are removing the check s to see if the button is enabled, I think is better to check them here.